scylla-artifacts.py

```diff
@@ -314,16 +317,30 @@ def run(self):
         if devlist:
             assert os.path.ismount('/var/lib/scylla'), "RAID setup failed, scylla directory isn't mounted rightly"
         # verify ntp
-        process.run('systemctl status ntpd')
+        if is_debian_variant:
+            process.run('service ntp status')
```

Comment:
Besides, Ubuntu adopted systemd not long ago. It's best to leave that sort of check to that library.